src/main/java/bookstore/dtos/book/CreateUpdateBookRequestDto.java

```diff
@@ -35,4 +36,6 @@ public class CreateUpdateBookRequestDto {
 
     @Length(min = 1, max = 255, message = "coverImage length must be between 1 and 255 characters")
     private String coverImage;
+
+    private List<Long> categories;
```

Comment:
Can the book exist without Category? 
```suggestion
    @NotEmpty
    private List<Long> categories;
```